packages/visx-drag/src/util/useSamplesAlongPath.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+
+function getSamples(restrictToPath: SVGGeometryElement, transform?: DOMMatrix, precision = 1) {
+  if (!restrictToPath) return [];
+  const samples = [];
+  const pathLength = restrictToPath.getTotalLength();
+  for (let sampleLength = 0; sampleLength <= pathLength; sampleLength += precision) {
+    const sample = restrictToPath.getPointAtLength(sampleLength);
```

Comment:
hmm, looks like the returned `DOMPoint` is [not supported in IE](https://developer.mozilla.org/en-US/docs/Web/API/DOMPointReadOnly), tho `.getPointAtLength` supposedly is [for `SVGPathElement`s](https://developer.mozilla.org/en-US/docs/Web/API/SVGGeometryElement/getPointAtLength) (not `SVGGeometryElement` generally) and I'm pretty sure we use it [elsewhere](https://github.com/airbnb/visx/blob/1ee35ff40b8e5f1de45bc613ce8e58770e5ff6aa/packages/visx-shape/src/util/getSplitLineSegments.ts#L61). also IE end of life is in June 2022 so maybe no need to worry about it 🎉